Makefile

```diff
@@ -204,16 +207,32 @@ $(BINODIR)/%.bin: $(IMAGEODIR)/%.out
 # Should be done in top-level makefile only
 #
 
-all:	sdk_pruned pre_build .subdirs $(OBJS) $(OLIBS) $(OIMAGES) $(OBINS) $(SPECIAL_MKTARGETS)
+all: toolchain sdk_pruned pre_build .subdirs $(OBJS) $(OLIBS) $(OIMAGES) $(OBINS) $(SPECIAL_MKTARGETS)
 
 .PHONY: sdk_extracted
 .PHONY: sdk_patched
 .PHONY: sdk_pruned
+.PHONY: toolchain
 
 sdk_extracted: $(TOP_DIR)/sdk/.extracted-$(SDK_BASE_VER)
 sdk_patched: sdk_extracted $(TOP_DIR)/sdk/.patched-$(SDK_VER)
 sdk_pruned: $(SDK_DIR_DEPENDS) $(TOP_DIR)/sdk/.pruned-$(SDK_VER)
 
+ifeq ($(OS),Windows_NT)
+toolchain:
```

Comment:
For Windows just ignore the pre-built toolchain.

Reply:
Ah, it occurs to me that I forgot to include something like `linux-x86_64` in the release bundle name. I had meant to do it, but then I was getting it all done in the background while working on other things and it slipped my mind. What's your view? Should I add it?

Reply:
I'm not aware of anyone building on Windows without using a Linux docker or such for the build. Maybe @marcelstoer knows more?

I was largely thinking whether there may be a case made for providing e.g. linux-aarch64, linux-armhf or even linux-x86 in the future. I could make it part of the directory name inside the archive easily enough in such cases (as well as the tar.xz name, naturally).

Reply:
Ah, that makes sense.

Reply:
I've never actually seen a Windows toolchain in the wild. Personally I wouldn't be sad if the Windows_NT ifeq was dropped. It's hinting at supporting something which I don't think has ever been tested. Should we ever need it back it's in the repo history anyway. Just my $.02, I'm not claiming to speak for the other core committers :)

Reply:
Alright, I'm testing a change that will change the filename to `toolchain-esp8266-linux-x86_64-<releasename>.tar.xz` and the internal directory from `esp8266` to `esp8266-linux-x86_64-<releasename>`. I think that should provide full co-existence support.

I wonder if I could get $work to get me a Threadripper 2990WX build box... compiling compilers is so tedious >.>

Reply:
Okay, it is [done](https://github.com/jmattsson/esp-toolchains/releases/tag/linux-x86_64-20181106.0).

Reply:
Everything should be up-to-date with your changes.

By the way it made things much nicer to have the directory in the tar be `esp8266-$(PLATFORM)-$(VERSION)/` instead of `esp8266/` so that `make` doesn't have to insert an extra layer :+1: